components/elasticsearch/scripts/create.py

```diff
@@ -238,6 +238,24 @@ def _install_elasticsearch():
     utils.systemd.enable(ES_SERVICE_NAME, append_prefix=False)
 
 
+def _wait_for_shards(port, ip):
+    ctx.logger.info('Waiting for shards to be active...')
+    shards_check_timeout = 60
+    start = time.time()
+    url = '{ip}:{port}/*/_search_shards'.format(ip=ip, port=port)
+    while True:
+        all_shards_started = True
+        proc = utils.run(['curl', '-XGET', url])
+        for shard in json.loads(proc.aggr_stdout)['shards']:
+            all_shards_started = all_shards_started and \
+                              (shard[0]['state'] == 'STARTED')
+        if all_shards_started:
+            return
+        time.sleep(1)
+        if time.time() - start > shards_check_timeout:
+            utils.error_exit('Elasticsearch shards check timeout')
```

Comment:
should be "timed out". Also, can we possible get any information about the reason from the rest call?